src/commands/publish.ts

```diff
@@ -65,10 +65,15 @@ export const publish = async (argv: string[]) => {
     return acc;
   }, [] as { name: string; errors: string[] }[]);
   if (invalidItemMessages.length > 0) {
-    console.error("Validation error:");
+    const chalk = (await import("chalk")).default;
     invalidItemMessages.forEach((msg) => {
-      console.error(msg.name, msg.errors);
+      msg.errors.forEach((err) => {
+        let text = chalk.red.bold(`${msg.name}: `);
+        text += chalk.red(err);
+        console.error(text);
```

Comment:
IMO&NITS
↓のような書き方の方が直感的に何をやっているかわかりやすいと感じました。

```suggestion
        const errorName = chalk.red.bold(msg.name);
        const errorDescription = chalk.red(err);
        console.error(`${errorName}: ${errorDescription}`)
```